### ISSUE TITLE
No visible binding for global variable "x" in dplyr syntax.

### ISSUE THREAD

**Issue by albert-ying** [r-lib/lintr#1540]
Hi -- It should be a common problem, but I couldn't find any solution anywhere:

I always get the "No visible binding for global variable" warning when doing something like:

```r

df |>
  mutate(a = b)

```

It will warn me that there is no visible binding for `b`. But the `b` is a column in `df`, and this code should be fine. Is there a way to disable this warning for the dplyr functions?

**Comment by IndrajeetPatil** [comment]
Hi, thanks for the report.

Unfortunately, this is not so much of a `{lintr}` issue, but a direct result of non-standard evaluation utilized in `{dplyr}` (and some other packages).

To see your options to get rid of this warning (which you will also see with `R CMD check`), see [this](https://nathaneastwood.github.io/2019/08/18/no-visible-binding-for-global-variable/) post.
